### ISSUE TITLE
训练自己的样本时map老是上不去。。。。请问有啥调参的建议不？

### ISSUE THREAD

**Issue by Ilikefoodverymuch** [bubbliiiing/faster-rcnn-keras#26]
看了下输出的结果发现recall非常低，即便precession已经降到0了，recall也只有80多，尝试了下调学习率（根本不太可能有用吧。。。。），没有效果，怀疑是RPN的问题，增加了RPN里两个loss的权重，也不行，请问还有别的优化方法吗？另外关于epoch_length参数是要设置成和自己的样本数量一致吗？如果我设置了一个小于样本数量的值a，就随机从我的样本中抽取a个进行训练？望各位不吝赐教/(ㄒoㄒ)/~~

**Comment by bubbliiiing** [comment]
训练的啥啊，map多少呀

**Comment by Ilikefoodverymuch** [comment]
训练的烟雾，通过修改现在已经提到96了，谢谢，不过还是想问下，读代码的时候关于resnet.py这里实现的BatchNormalization类，这个类里的call和build方法并没有找到有显式调用的地方，请问是他们起啥作用啊？call的话是不是和__call__()一样呢？build我真没找到资料/(ㄒoㄒ)/~~
>

**Comment by chenjjjjuu** [comment]
你好，我训练自己的数据集map也上不去，请问你是如何解决的/

**Comment by bubbliiiing** [comment]
BatchNormalization只是keras的BatchNormalization冻结权重后的一个自定义层，因为batch=1，不冻结不能用

**Comment by chenjjjjuu** [comment]
嗯嗯，请问我要如何改？



---原始邮件---
发件人: "Bubbliiiing"<notifications@github.com&gt;
发送时间: 2020年9月12日(周六) 中午12:17
收件人: "bubbliiiing/faster-rcnn-keras"<faster-rcnn-keras@noreply.github.com&gt;;
抄送: "Comment"<comment@noreply.github.com&gt;;"chenjjjjuu"<1540358700@qq.com&gt;;
主题: Re: [bubbliiiing/faster-rcnn-keras] 训练自己的样本时map老是上不去。。。。请问有啥调参的建议不？ (#26)




 
BatchNormalization只是keras的BatchNormalization冻结权重后的一个自定义层，因为batch=1，不冻结不能用

**Comment by bubbliiiing** [comment]
就默认参数呀，我的模型就是默认参数训练的
